File: 프로그래머스/1/340000/340199. ［PCCE 기출문제］ 9번 ／ 지폐 접기/［PCCE 기출문제］ 9번 ／ 지폐 접기.py
def solution(wallet, bill):
    answer = 0
    while max(wallet) < max(bill) or min(bill) > min(wallet) :
        bill[bill.index(max(bill))] = max(bill)//2
        answer += 1  # 접는 횟수 증가

    return answer

File: 프로그래머스/1/340000/340199. ［PCCE 기출문제］ 9번 ／ 지폐 접기/［PCCE 기출문제］ 9번 ／ 지폐 접기.java
class Solution {
    public int solution(int[] wallet, int[] bill) {
        int answer = 0;
        while (Math.max(wallet[0], wallet[1]) < Math.max(bill[0], bill[1]) ||
               Math.min(wallet[0], wallet[1]) < Math.min(bill[0], bill[1])) {
            
            // bill 배열에서 더 큰 값을 찾아 반으로 줄이기
            if (bill[0] > bill[1]) {
                bill[0] /= 2;
            } else {
                bill[1] /= 2;
            }
            
            answer++;  // 접는 횟수 증가
        }
        return answer;
    }
}